config/settings.py

```diff
@@ -34,62 +34,60 @@
 
 INSTALLED_APPS = [
```

Comment:
You can divide this list into three:-
1. DJANGO_APPS = [ ]
2. PACKAGE_APPS = [ ]
3. CUSTOM_APPS = [ ]
then,
INSTALLED_APPS = DJANGO_APPS +  PACKAGE_APPS + CUSTOM_APPS